src/BlockTemplatesController.php

```diff
@@ -67,7 +67,7 @@ protected function init() {
 		add_filter( 'get_block_templates', array( $this, 'add_block_templates' ), 10, 3 );
 		add_filter( 'current_theme_supports-block-templates', array( $this, 'remove_block_template_support_for_shop_page' ) );
 		add_filter( 'taxonomy_template_hierarchy', array( $this, 'add_archive_product_to_eligible_for_fallback_templates' ), 10, 1 );
-
+		add_filter( 'woocommerce_product_tabs', 'woocommerce_default_product_tabs' );
```

Comment:
Is it necessary? 🤔

Reply:
You're correct, this is not necessary. I forgot to remove this piece of code that was put during my code investigation 😞